mindmeld/components/entity_resolver.py

```diff
@@ -75,6 +75,7 @@ def __init__(
         self._er_config = get_classifier_config("entity_resolution", app_path=app_path)
         self._es_host = es_host
         self._es_config = {"client": es_client, "pid": os.getpid()}
+        self._already_fit = False
```

Comment:
is this equivalent to `ready` in the classifiers?

Reply:
Yes, its the same, except we actually use the flag here as a conditional, while we dont use `ready` for the models for a practical purpose. I will match the name.